fe/fe-core/src/main/java/com/starrocks/leader/ReportHandler.java

```diff
@@ -1183,6 +1227,11 @@ public static void testHandleSetTabletBinlogConfig(long backendId, Map<Long, TTa
         handleSetTabletBinlogConfig(backendId, backendTablets);
     }
 
+    public static void testHandleMigration(ListMultimap<TStorageMedium, Long> tabletMetaMigrationMap,
```

Comment:
You can change the signature of handleMigration from `private` to `protected`, and this function can be removed. Function only for test is not recommended

Reply:
done